lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -169,7 +165,11 @@ def eci_value(payment_method, options)
       end
 
       def authorize_request(money, payment_method, options)
-        commit('authorize', build_authorization_request(money, payment_method, options), 'AUTHORISED', 'CAPTURED', options)
+        if options[:account_funding_transaction]
+          commit('funding_transfer_transaction', build_aft_request(money, payment_method, options), :ok, 'AUTHORISED', options)
```

Comment:
Can we point this to `aft_request(money, payment_method, options)` instead? That way if any changes need to be made to how the `build_aft_request` method is referenced, it's not repeated in more than one place

Reply:
Makes sense, done!